.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  pull_request:
+
+env:
+  FORCE_COLOR: 2
+  NODE: 14
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "${{ env.NODE }}"
+          cache: npm
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Test
+        run: npm test
+
+      - name: Upload docs
+        uses: actions/upload-artifact@v2
+        if: github.repository == 'jakearchibald/svgomg' && github.ref == 'refs/heads/main'
+        with:
+          name: docs
+          path: ./build/
+          if-no-files-found: error
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.repository == 'jakearchibald/svgomg' && github.ref == 'refs/heads/main'
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Download docs
+        uses: actions/download-artifact@v2
+        with:
+          name: docs
+          path: ./build/
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          allow_empty_commit: false
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This should be enough IIRC. If not, we could use a PAT later.

Reply:
Where will this be set?

Reply:
`GITHUB_TOKEN` is set by default https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

If we go with PAT, then in the repo settings -> Secrets.